lib/main.js

```diff
@@ -43,6 +49,17 @@ var runSass = function(sassOptions, trueOptions) {
   } else {
     sassOpts.includePaths = [sassPath];
   }
+
+  if (sassOpts.file) {
+    sassOpts.data = fs.readFileSync(sassOpts.file, 'utf-8');
+    sassOpts.includePaths.push(path.dirname(sassOpts.file));
+    delete sassOpts.file;
+  }
+
+  // remove comments in case someone commented an error assertion
+  sassOpts.data = sassOpts.data.replace(REG_COMMENT, '$1');
```

Comment:
This is done because we are doing simple string parsing to detect any error assertions. Each error assertion is then "activated" one at a time inside a try/catch block - and we expect to actually catch an error. If no error is thrown, then it is reported as a failure. Therefore, we must strip any comments which might be wrapped around an error assertion. This is not ideal, but I cannot justify bringing in a full parser.